src/applications/personalization/account/components/MultifactorMessage.jsx

```diff
@@ -1,36 +1,54 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { ssoe } from '../../../../platform/user/authentication/selectors';
```

Comment:
These can all be simplified to something like `'platform/user/authentication/selectors'`.

Reply:
Likewise in other places if it's not too much of a change. Can import from `'platform/...'` as a relative path.

Reply:
will do